demos/sjtwo/eeprom/source/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include <project_config.hpp>
+
+#include <cstdint>
+#include <iterator>
+
+#include "L1_Peripheral/lpc40xx/eeprom.hpp"
+#include "L2_HAL/displays/led/onboard_led.hpp"
+#include "utility/log.hpp"
+#include "utility/time.hpp"
+
+int main(void)
+{
+  LOG_INFO("Starting EEPROM Example");
+  constexpr size_t kPayloadSize = 128;
+  constexpr uint32_t kAddress   = 0b0110'0011'1000;
+
+  uint8_t * results = new uint8_t[kPayloadSize];
```

Comment:
https://sjsu-dev2.readthedocs.io/en/latest/rst/contributing/style.html#memory-allocation

Refrain from using dynamic memory.
```suggestion
  uint8_t results[kPayloadSize];
```